src/main/groovy/com/github/jrubygradle/jar/JRubyJarConfigurator.groovy

```diff
@@ -12,8 +12,8 @@ import com.github.jrubygradle.GemUtils
  */
 class JRubyJarConfigurator {
 
-    static final String DEFAULT_BOOTSTRAP_CLASS = 'com.github.jrubygradle.jar.bootstrap.JarMain'
-    static final String SHADOW_JAR_TASK_CLASS = 'com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar'
+    static final String DEFAULT_MAIN_CLASS = 'de.saumya.mojo.mains.JarMain'
+    static final String DEFAULT_EXTRACTING_MAIN_CLASS = 'de.saumya.mojo.mains.ExtractingMain'
```

Comment:
I think you read my comment in the meanwhile. sometimes running from inside the jar fails like "rake spec" using RSpec rake task does spawn ruby and this spawning will fail. will see to JRuby itself to fix it on the JRuby if possible. but for the time being we need to offer both cases to cover more applications (hopefully all applications)